#!/usr/bin/python
PACKAGE = 'recognition'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('model_folder_path',     str_t,   0, 'the path to the face detector modelfile', '/data/object_detector')
gen.add('confidence_thresh',      double_t, 0, 'the threshold for face detection confidence', 0.3, -2.0, 2.0)

exit(gen.generate(PACKAGE, 'recognition', 'GenDetection'))
